iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/TreeFactory.java

```diff
@@ -46,11 +48,27 @@ public TargetScopeDeclaration targetScopeDeclaration(SyntaxToken targetScope, Sy
     return new TargetScopeDeclarationImpl(targetScope, equals, expression);
   }
 
+  public VariableDeclaration variableDeclaration(SyntaxToken variableKeyword, Identifier identifier, SyntaxToken equals, Expression expression) {
```

Comment:
I would suggest renaming `variableKeyword` to `keyword` and using a `keyword` in this class and in every Bicep Tree implementation. We did that in Docker and it worked well.